.travis.yml

```diff
@@ -18,7 +18,9 @@ script:
 - cd ..;
 - travis-cargo build
 - travis-cargo test
+- travis-cargo test -- --features=deadlock_detection
 - travis-cargo doc -- --no-deps -p parking_lot -p parking_lot_core
+- travis-cargo --only nightly doc -- --all-features --no-deps -p parking_lot -p parking_lot_core
```

Comment:
Why was this line added?

Reply:
Ahh, it should have replaced the one above

Reply:
It's to build the docs including all features, otherwise deadlock_detection will not show.

Reply:
Then just add `--all-features` to the cargo doc line above.